ui/src/shared/utils/downloadTimeseriesCSV.ts

```diff
@@ -27,9 +26,20 @@ const downloadTimeseriesCSV = async (
   downloadCSV(csv, csvName())
 }
 
+export const downloadFluxCSV = async (
+  service: Service,
+  script: string
+): Promise<{didTruncate: boolean}> => {
+  const {csv, didTruncate} = await executeFluxQuery(service, script)
```

Comment:
any concern if it fails?

Reply:
I think it's appropriate for the caller to handle errors here (like above with the `try` / `catch` block)